dev-packages/e2e-tests/test-applications/nextjs-14/package.json

```diff
@@ -13,17 +13,17 @@
     "test:assert": "pnpm test:prod && pnpm test:dev"
   },
   "dependencies": {
+    "@playwright/test": "^1.27.1",
     "@sentry/nextjs": "latest || *",
     "@types/node": "18.11.17",
     "@types/react": "18.0.26",
     "@types/react-dom": "18.0.9",
-    "next": "14.0.4",
+    "next": "14.1.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "4.9.5",
-    "wait-port": "1.0.4",
     "ts-node": "10.9.1",
-    "@playwright/test": "^1.27.1"
+    "typescript": "4.9.5",
+    "wait-port": "1.0.4"
```

Comment:
just to clarify, are these new deps important/needed?

Reply:
They're not new. Yarn just auto sorted them. The only thing I did was bump the version the default e2e tests run with to the latest version. I think that is needed. At least it was to verify this bug.